aptiche/lib/views/quiz/quiz_controller.dart

```diff
@@ -35,6 +35,8 @@ class QuizController extends GetxController {
   ///Stores the user score of the quiz
   Rx<int> userScore = 0.obs;
 
+  ///[questionIndex] stream subscription
+  StreamSubscription<int>? _questionIndexStream;
```

Comment:
@SriramPatibanda is stream a right choice or RxInt should be used?

Reply:
Stream can be used.

Reply:
I used stream as rxint updates the UI when a diffrent value is assigned to it .To solve this either I can use a getBuilder or create a temporary stream subscription to update the UI when same value is passes to it.